pkg/controller/data_download_controller.go

```diff
@@ -182,7 +182,7 @@ func (r *DataDownloadReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 		hostingPodLabels := map[string]string{velerov1api.DataDownloadLabel: dd.Name}
 		for _, k := range util.ThirdPartyLabels {
-			if v, err := nodeagent.GetLabelValue(ctx, r.kubeClient, dd.Namespace, k); err != nil {
+			if v, err := nodeagent.GetLabelValue(ctx, r.kubeClient, dd.Namespace, k, kube.NodeOSLinux); err != nil {
```

Comment:
Why the OS is hardcoded as `Linux` here while reading OS from the PVC attaching node in data upload controller?

Reply:
See https://github.com/vmware-tanzu/velero/pull/8555#discussion_r1900506986